indicators/Smma/README.md

```diff
@@ -0,0 +1,55 @@
+# Smoothed Moving Average (SMMA)
+
+[Smoothed Moving Average](https://en.wikipedia.org/wiki/Moving_average#Modified_moving_average) is the average of Close price over a lookback window using a smoothing method.
+[[Discuss] :speech_balloon:](https://github.com/DaveSkender/Stock.Indicators/discussions/375 "Community discussion about this indicator")
+
+![image](chart.png)
```

Comment:
Also, minimum history is probably more due to convergence errors / slow warmup.

Reply:
Since SMMA is a variant of EMA, my instinct is to copy the EMA's minimum history requirements of N * 2. Not sure how to test convergence errors or slow warmup, which I'm guessing is why the requirement is `You must supply at least `2×N` or `N+100` periods of `history`, whichever is more.`?

Reply:
Here's some convergence profiling using S&P data.  Speed to converge is both a function of lookback period and the actual data that drives the initial SMA starting point.  If the initial SMA starting point is way off, it takes longer to get back to where it's supposed to be.

```
SMMA(5) on 09/04/2020 with    5 periods: 3,497.96400000
SMMA(5) on 09/04/2020 with   20 periods: 3,473.46490977
SMMA(5) on 09/04/2020 with   30 periods: 3,472.84234001
SMMA(5) on 09/04/2020 with   50 periods: 3,472.82533688
SMMA(5) on 09/04/2020 with   75 periods: 3,472.82592174
SMMA(5) on 09/04/2020 with  100 periods: 3,472.82591833
SMMA(5) on 09/04/2020 with  120 periods: 3,472.82591831
SMMA(5) on 09/04/2020 with  150 periods: 3,472.82591831
SMMA(5) on 09/04/2020 with  200 periods: 3,472.82591831
SMMA(5) on 09/04/2020 with  250 periods: 3,472.82591831
SMMA(5) on 09/04/2020 with  350 periods: 3,472.82591831
SMMA(5) on 09/04/2020 with  500 periods: 3,472.82591831
SMMA(5) on 09/04/2020 with  600 periods: 3,472.82591831
SMMA(5) on 09/04/2020 with  700 periods: 3,472.82591831
SMMA(5) on 09/04/2020 with  800 periods: 3,472.82591831
SMMA(5) on 09/04/2020 with  900 periods: 3,472.82591831
SMMA(5) on 09/04/2020 with 1000 periods: 3,472.82591831

SMMA(15) on 09/04/2020 with   15 periods: 3,451.02000000
SMMA(15) on 09/04/2020 with   30 periods: 3,409.33484819
SMMA(15) on 09/04/2020 with   40 periods: 3,401.77761249
SMMA(15) on 09/04/2020 with   60 periods: 3,394.76005103
SMMA(15) on 09/04/2020 with   85 periods: 3,393.81357414
SMMA(15) on 09/04/2020 with  110 periods: 3,393.49017533
SMMA(15) on 09/04/2020 with  130 periods: 3,393.46149688
SMMA(15) on 09/04/2020 with  160 periods: 3,393.52152382
SMMA(15) on 09/04/2020 with  210 periods: 3,393.52059552
SMMA(15) on 09/04/2020 with  260 periods: 3,393.52054926
SMMA(15) on 09/04/2020 with  360 periods: 3,393.52054894
SMMA(15) on 09/04/2020 with  510 periods: 3,393.52054894
SMMA(15) on 09/04/2020 with  610 periods: 3,393.52054894
SMMA(15) on 09/04/2020 with  710 periods: 3,393.52054894
SMMA(15) on 09/04/2020 with  810 periods: 3,393.52054894
SMMA(15) on 09/04/2020 with  910 periods: 3,393.52054894
SMMA(15) on 09/04/2020 with 1010 periods: 3,393.52054894

SMMA(150) on 09/04/2020 with  155 periods: 3,059.37247903
SMMA(150) on 09/04/2020 with  170 periods: 3,082.38064932
SMMA(150) on 09/04/2020 with  180 periods: 3,093.71849221
SMMA(150) on 09/04/2020 with  200 periods: 3,105.59468321
SMMA(150) on 09/04/2020 with  225 periods: 3,109.48292269
SMMA(150) on 09/04/2020 with  250 periods: 3,104.08001121
SMMA(150) on 09/04/2020 with  270 periods: 3,095.05978111
SMMA(150) on 09/04/2020 with  300 periods: 3,085.80479320
SMMA(150) on 09/04/2020 with  350 periods: 3,073.53653949
SMMA(150) on 09/04/2020 with  400 periods: 3,066.19535281
SMMA(150) on 09/04/2020 with  500 periods: 3,056.42494259
SMMA(150) on 09/04/2020 with  650 periods: 3,056.78244604
SMMA(150) on 09/04/2020 with  750 periods: 3,055.83188229
SMMA(150) on 09/04/2020 with  850 periods: 3,054.61401767
SMMA(150) on 09/04/2020 with  950 periods: 3,054.17606331
SMMA(150) on 09/04/2020 with 1050 periods: 3,053.91976666
SMMA(150) on 09/04/2020 with 1150 periods: 3,053.81595496
SMMA(150) on 09/04/2020 with 1400 periods: 3,053.80701215
SMMA(150) on 09/04/2020 with 1650 periods: 3,053.79661892
```